### DISCUSSION TITLE
Problem on Win 10 (Microsoft Surface Go2)

### DISCUSSION THREAD

**Discussion by 68pandreas** [xournalpp/xournalpp#2485] [Q&A]
Hi, I came accross Xournal and found it very interesting. It seams to have however problems on my Win 10 calculator. The Pen does not seem to work properly and in input devices it is not listed. There is only one input device, which seams to be the touch screen. Are there known problems? Will this be somehow fixed? Are further details needed? Best regards, Andreas

**Comment by LittleHuba** [comment]
That is unfortunately a known issue of GTK. Windows devices are merged together to one single OS pointer device. Nothing we can do about that.

**Reply by 68pandreas** [reply]
Thanks for the quick reply! Is there further information at GTK and are they aware of the problem? Icould fancy that this would be not only an issue for Xournal++, would it?

**Reply by LittleHuba** [reply]
This is definitely not only an issue of Xournal++ but all applications using GTK in the background (GIMP, Inkscape,...) running on Windows.

You can check out #2316 where other users with Win10 have posted their experiences with Xounral++ using the updated GTK. Please be aware that all this is still a work in progress and there are still some bugs. But this is at least a starting point. The pointer devices still seem to be aggregated though... I assume this is more a restriction in Windows than in the GTK code.

**Reply by LittleHuba** [reply]
https://gitlab.gnome.org/GNOME/gtk/-/issues/1845

This is another ticket that mentiones the issue of the system aggregated pointer. So GTK seems possible to see the actual devices but not consistently.

**Comment by 68pandreas** [comment]
Hi, meanwhile I found out about the reason: wintab32.dll was missing on my Surface Go2. I've installed Wintab_x64_1.0.0.20.zip downloaded from https://www.microsoft.com/en-us/download/details.aspx?id=49498 (actually for the Surface Pro 4) and it seams to work, at least I get now a mouse and a stylus. I also found that there is another free tool named writer300 from styluslabs.com which interestingly works correct without wintab. The problem with wintab may be that at any point in time it may be not supported anylonger by Microsoft as it seems to be the case already for the Surface Go2. Maybe it might make sense to add an hint for the Windows releases that wintab32.dll needs to be present in the system. Best regards, Andreas

**Reply by LittleHuba** [reply]
You are welcome to provide a PR for our Website and the documentation in this repository including such notices. As GTK devs are already working on inclusion of the newer Windows Pointer API this problem should fix itself in the long run.

**Reply by 68pandreas** [reply]
Sorry, I am not really in the Github community and don't know what a PR is and how it is done in detail. With respect to content the windows download link could contain a small hint like "Attention: wintab32.dll needed for full functionality!" Best regards Andreas
